third_party/googletest/src/googletest/include/gtest/internal/gtest-port.h

```diff
@@ -2086,7 +2087,7 @@ inline int StrCaseCmp(const char* s1, const char* s2) {
 #endif //SB_API_VERSION < 16
 inline char* StrDup(const char* src) { return strdup(src); }
 
-inline int RmDir(const char* dir) { return SbFileDelete(dir); }
+inline int RmDir(const char* dir) { return rmdir(dir); }
```

Comment:
Should this be `return rmdir(dir) == 0;`?

Reply:
I assume the behavior of RmDir should be the same as posix rmdir (it also has the same type). And I looked at the original google3 implementation and chrome implementation, both seem to use `return rmdir(dir);` directly.

Reply:
Nice!